AstronomCore/src/main/java/com/astronomvm/core/model/database/JDBCDriver.java

```diff
@@ -0,0 +1,20 @@
+package com.astronomvm.core.model.database;
+
+import lombok.Data;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'lombok.Data'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ahmedriahi-github/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=AhmedRiahi_AstronomVM&pullRequest=22&issues=AWZfQr2YDZywvPe4Lfue&open=AWZfQr2YDZywvPe4Lfue)

[ISSUE_KEY]: AWZfQr2YDZywvPe4Lfue